apps/chat/src/contexts/ChatContext.tsx

```diff
@@ -1,12 +1,12 @@
 import { createContext, useContext, ReactNode, useState } from "react";
-import { DeserializedSearchResult } from "../view/types";
+import { SearchResultWithSnippet } from "../view/types";
```

Comment:
These are generally the same data types, except that `SearchResultWithSnippet` references properties from search results returned by stream query (e.g. `document_id` instead of just `id`). 

The main reason for renaming is that this data type is no longer the result of deserializing anything, and IMO having a `DeserializedSearchResult` data type leads maintainers to mistakenly believe we're still deserializing search results like before.

Reply:
Makes sense!